be/src/util/block_compression.cpp

```diff
@@ -183,17 +183,14 @@ class Lz4fBlockCompression : public BlockCompressionCodec {
 };
 
 LZ4F_preferences_t Lz4fBlockCompression::_s_preferences = {
-        {
-                LZ4F_max256KB,
-                LZ4F_blockLinked,
-                LZ4F_noContentChecksum,
-                LZ4F_frame,
-                0,     // unknown content size,
-                {0, 0} // reserved, must be set to 0
-        },
-        0,            // compression level; 0 == default
-        0,            // autoflush
-        {0, 0, 0, 0}, // reserved, must be set to 0
+        {LZ4F_max256KB, LZ4F_blockLinked, LZ4F_noContentChecksum, LZ4F_frame,
```

Comment:
The interface has been changed. So we should add some case to test compatibility.